roles/redpanda_broker/templates/console/defaults.j2

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
We should add the schema registry config here too

Reply:
```yaml
kafka:
  brokers: []
  schemaRegistry:
    enabled: true
    urls: [""]
  protobuf:
    enabled: true
    schemaRegistry:
      enabled: true
```

This is what we should add and prefill the urls accordingly.

Reply:
Startup will fail, so should only be configured if also configured in RP

Reply:
Ok, then I'll leave it out by default but include a note about how to add w/ the rpconsole variable.

Reply:
Is there a reason to not always enable the schema registry on Redpanda? Can this be explicitly opted out from in the ansible role?

Reply:
Based on some additional testing I did today, setting it up properly requires SASL/ACL which are a future goal for the ansible project but aren't currently ready to go. Getting all that together would be a pretty significant scope creep for this PR.